commands/rm.go

```diff
@@ -31,14 +31,15 @@ func cmdRm(c *cli.Context) {
 	}
 
 	for _, host := range c.Args() {
+		log.Infof("Running action %q on machine %q...", "remove", host)
 		if err := mcn.Remove(host, force); err != nil {
-			log.Errorf("Error removing machine %s: %s", host, err)
+			log.Errorf("Error running action %q on machine %q: %s", "remove", host, err)
 			isError = true
 		} else {
 			log.Infof("Successfully removed %s", host)
 		}
 	}
 	if isError {
-		log.Fatal("There was an error removing a machine. To force remove it, pass the -f option. Warning: this might leave it running on the provider.")
+		log.Fatal("There was an error removing one or more machines.")
```

Comment:
This change may or may not be a little controversial, but I'm mostly of the opinion we shouldn't really be tipping users off to / encouraging the use of `-f` -- and we should warn that it may still be running on the provider there, not here as a hypothetical.